### DISCUSSION TITLE
Feature Request: Discussions to support auto-linking to/from GitHub Issues / PRs when containing link references to each other in comments

### DISCUSSION THREAD

**Discussion by HariSekhon** [community/community#19572] [Discussions]
GitHub Issues and Pull Requests can automatically show references to other tickets mentioning them.

This is a Feature Request for discussions to do the same and cross-link with issues in repositories that are referencing it and vice versa, so that both discussions and issues automatically get links to each other when referenced.

Use Case:

- better clarity of linked use case issues
- better documentation cross-referencing
- more real-world data on how what is are affected by an issue, if they have issues in repos that reference being blocked on this, it gives developers and product managers a better understanding of the implications of this discussion / feature request.

For example, the following tickets:

https://github.com/integrations/terraform-provider-github/issues/1199

https://github.com/integrations/terraform-provider-github/issues/1200

reference these discussions:

https://github.com/orgs/github-community/discussions/19569

https://github.com/orgs/github-community/discussions/19570

but neither of them have not gotten the auto-links that usually appear when one issue or PR references another.

**Comment by danielorbach** [comment]
Last I heard about it was in #2971 where some GitHub employees updated they're working on it.
